examples/gno.land/r/demo/gnolotto_factory/README.md

```diff
@@ -0,0 +1,405 @@
+# Write a simple Lottery on Gno.land
+
+## Overview
+
+This guide will demonstrate how to write a simple lottery on GnoLand. We'll cover adding funds to the realm, buying tickets for participation, and finally, distributing the winnings once the winning numbers are drawn. Each step is designed to ensure a smooth, transparent process from the lottery's inception to the awarding the prizepool.
```

Comment:
When writing .md files, you should break lines at about 80 characters, so that they  fit in screens when reading them raw.